### DISCUSSION TITLE
Language support

### DISCUSSION THREAD

**Discussion by Guray00** [streetwriters/notesnook#865] [Ideas]
Are you planning to add a language platform as support to translation (by users)? Something like crowdin

**Comment by shuvashish76** [comment]
I'd suggest [Weblate](https://github.com/WeblateOrg/weblate) which is FLOSS as well.

Features: https://weblate.org/en/features/
![Screenshot_2021-12-05-21-15-09](https://user-images.githubusercontent.com/48226312/144753433-4fc61e84-f12d-42d0-885e-1b9abd399061.jpeg)
https://weblate.org/en/hosting/

Not in favour of proprietary software Crowdin.
Compare both their privacy policy yourself:
 - https://support.crowdin.com/privacy-policy/
 - https://weblate.org/en/terms/

![image](https://user-images.githubusercontent.com/48226312/188296551-c10b7890-8490-47fe-bfae-b24d7af94429.png)
As @comradekingu mentioned somewhere Crowdin is _not_ for opensource projects, since it specifically focus on non-profit opensource projects which means it requires projects to have no source of income, or profit motive. _(see [criteria no4](https://crowdin.com/page/open-source-project-setup-request))_

---

I'm actually impressed by Notesnook privacy related blog posts e.g. [Notesnook & email](https://blog.notesnook.com/why-notesnook-requires-an-email-address), [DuckDuckGo](https://blog.notesnook.com/is-duckduckgo-search-browser-really-private) related, [2FA](https://blog.notesnook.com/improving-user-account-security-with-2fa), [WhatsApp](https://blog.notesnook.com/privacy-on-whatsapp-is-a-joke) related.
IMO for such privacy centric software it doesn't sound good to use proprietary software when we already have opensource alternative.

**Comment by jermanuts** [comment]
https://github.com/streetwriters/notesnook/issues/951 hope they support it anytime soon 😞

**Comment by cuhsy** [comment]
Seems like this has been forgotten. I hope they can reconsider, I personally could really use Greek language support inside the app, even if that meant I wrote the translation myself.

**Reply by thecodrr** [reply]
We are working on localization in #6231

**Reply by cuhsy** [reply]
> We are working on localization in #6231

What platform will be used for contributors that want to help translate?

**Comment by dyw770** [comment]
Hello, the localization branch has been merged into the master, how can I contribute translation?
